<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wifi Mode Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionManaged_mode"/>
    <addaction name="actionMonitor_mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionMonitor_mode"/>
   <addaction name="actionManaged_mode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMonitor_mode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Monitor mode</string>
   </property>
  </action>
  <action name="actionManaged_mode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Managed mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
